package/base-files/files/lib/functions/uci-defaults.sh

```diff
@@ -617,6 +617,52 @@ ucidef_set_hostname() {
 	json_select ..
 }
 
+ucidef_set_wireless() {
+	local ssid="$1"
+	local encryption="$2"
+	local key="$3"
+
+	json_select_object wireless
+		json_add_string ssid "$ssid"
+		[ -n "$encryption" ] && json_add_string country "$encryption"
+		[ -n "$key" ] && json_add_string key "$key"
+	json_select ..
+}
+
+ucidef_enable_wireless() {
+	local bands="$1"
+	local enable_2ghz
+	local enable_5ghz
+
+	case "$bands" in
```

Comment:
Please rework this to follow the _bands_ defined in [mac80211.sh](https://github.com/openwrt/openwrt/pull/4777/commits/1fb7a98af010832538c1a2fe8a6c6fb0ac5d9ba0#diff-02d22d38e6014fd3288251e872250d5763063957ba37740831f19b6003b16a4a)